.github/workflows/test-report.yml

```
@@ -0,0 +1,20 @@
+﻿name: Report
+
+on:
+  workflow_run:
+    workflows: ['Build and Test']
```

```suggestion
    workflows: [Build and Test]
```

[Based on the samples](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run) you don't need the apostrophes. Could it be that this workflow is now waiting for a workflow called "'Build and Test'"?

---

It's waiting for the workflow already present yes.

---

It's strating after the whole workflow is done, so after all 4 have either failed or succeeded.